### PR TITLE
Refactor responsiveness

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
Responsiveness is taken care of and also the routes been refactored

### CODE REVIEW

File: src/components/DifferentCards/Cart-Product-Big/DescriptionCard.jsx [hunk 1]
```diff
@@ -108,7 +108,6 @@ function CardProductBig({
                     </span>
                   </div>
                   <div className="product-rating text">
-                   
                     <Rating rating={parseInt(wholeItem.rating)} />
```
Comment (aayushjain-code) [inline src/components/DifferentCards/Cart-Product-Big/DescriptionCard.jsx:111]: you can create separate function for this functionality

File: src/index.css [hunk 2]
```diff
@@ -263,4 +265,22 @@ html::-webkit-scrollbar-track {
 }
 .Toastify{
   font-size: 16px;
```
Comment (aayushjain-code) [inline src/index.css:267]: you can use rem values

File: src/pages/cart-page.jsx/cart-page.css [hunk 3]
```diff
@@ -19,4 +19,16 @@
...
     height: 45rem;
     background-color: var(--PURE-WHITE);
+}
+@media screen and (max-width: 800px) {
+.carts-page{
+    padding: 1rem;
+    display: grid;
+    /* grid-template-rows:1fr 8fr 1fr; */
+    grid-template-columns:45rem 8fr ;
+    /* grid-template-areas: "... item-display item-display item-display ... payment-card "; */
```
Comment (aayushjain-code) [inline src/pages/cart-page.jsx/cart-page.css]: you can delete commented code

**Review by aayushjain-code — APPROVED** [review]
Overall code looks fine with a few suggestions you can merge it✅

File: src/components/ResponsiveAside/ResponsiveAside.css [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+.aside-container-pri h2 {
+    border-bottom: 2px solid var(--GREY-DARK);
+}
+
+
+
+
+.products-main {
+    /* border:solid red; */
```
Comment (sainath7878) [inline src/components/ResponsiveAside/ResponsiveAside.css]: remove commented code

**Review by sainath7878 — APPROVED** [review]
Everything looks good. Just remove commented out code and you're good to merge!

File: src/components/ResponsiveAside/ResponsiveAside.css [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    margin-left: 2rem;
+}
+
+.aside-container-pri h2 {
+    border-bottom: 2px solid var(--GREY-DARK);
+}
+
+
+
+
```
Comment (devsurajrai) [inline src/components/ResponsiveAside/ResponsiveAside.css:39]: take care of the spacing between the codes.

**Review by devsurajrai — COMMENTED** [review]
Good Work. Just take care of the spacing between the codes.

File: src/index.css (hunk 2 above)
Comment (Dev-AkshatGupta) [reply src/index.css:267]: actually that's a static size don't need that's why did so there

File: src/pages/cart-page.jsx/cart-page.css (hunk 3 above)
Comment (Dev-AkshatGupta) [reply src/pages/cart-page.jsx/cart-page.css]: yeah I will do that

File: src/components/ResponsiveAside/ResponsiveAside.css (hunk 4 above)
Comment (Dev-AkshatGupta) [reply src/components/ResponsiveAside/ResponsiveAside.css]: yeah gonna do that thanks for reviewing

File: src/components/ResponsiveAside/ResponsiveAside.css (hunk 5 above)
Comment (Dev-AkshatGupta) [reply src/components/ResponsiveAside/ResponsiveAside.css:39]: Ok  I will rectify it

**Pull request merged**